src/test/resources/org/broadinstitute/hellbender/tools/picard/fasta/testFASTA_WS_4.fasta

```diff
@@ -0,0 +1,284 @@
+>Chromosome
```

Comment:
create a directory `src/test/resources/org/broadinstitute/hellbender/tools/picard/fasta/NormalizeFastaIntegration`
and move this file there. And change the code in NormalizeFastaIntegrationTest.java accordingly (ie change "picard/fasta/"  to "picard/fasta/NormalizeFastaIntegration" on line 17)